projects/cm_mcu/commands/SensorControl.c

```diff
@@ -307,13 +307,12 @@ BaseType_t adc_ctl(int argc, char **argv, char *m)
   return pdFALSE;
 }
 
-// this command takes up to two arguments
-BaseType_t ff_ctl(int argc, char **argv, char *m)
```

Comment:
we have lost a lot of functionality here that was previously there in `ff_ctl` and that was heavily used...